internal/service/service.go

```diff
@@ -149,13 +175,13 @@ func (s *service) SavePDV(ctx context.Context, p schema.PDV, owner sdk.AccAddres
 		return 0, PDVMeta{}, fmt.Errorf("failed to write meta meta to storage: %w", err)
 	}
 
-	if meta.Reward > 0 {
+	if meta.Reward.IsPositive() {
 		log.WithFields(logrus.Fields{
 			"owner":  owner.String(),
 			"meta":   id,
```

Comment:
Why  did we decide to remove ID from the DistributeReward operations?

Reply:
it's not used anywhere

Reply:
I believe we stopped using that when we moved stats functionality to the Theseus service